js.go

```diff
@@ -359,6 +361,17 @@ func DirectGetNext(subject string) JSOpt {
 	})
 }
 
+// StreamListFilter is an option that can be used to configure `StreamsInfo()` and `StreamNames()` requests.
+// It allows filtering the retured streams by subject associated with each stream.
+// Wildcards can be used. For example, `StreamListFilter(FOO.>.A) will return
```

Comment:
`FOO.>.A` is an invalid subject I think, maybe switch to the asterisk wildcard variant:
```suggestion
// Wildcards can be used. For example, `StreamListFilter(FOO.*.A) will return
```

Reply:
Thanks for catching that - fixed